CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## develop
+- `PlaybackToggleButton` now also listens to ON_PLAYING in addition of ON_PLAY
```

Comment:
* Please adhere to the CHANGELOG format and put the entry under the "Changed" subsection
 * Put event names into `code` tags
 * Grammar: "in addition to"